website/content/en/preview/reference/settings.md

```diff
@@ -28,15 +26,15 @@ Karpenter surfaces environment variables and CLI parameters to allow you to conf
 | KARPENTER_SERVICE | \-\-karpenter-service | The Karpenter Service name for the dynamic webhook certificate|
 | KUBE_CLIENT_BURST | \-\-kube-client-burst | The maximum allowed burst of queries to the kube-apiserver (default = 300)|
 | KUBE_CLIENT_QPS | \-\-kube-client-qps | The smoothed rate of qps to kube-apiserver (default = 200)|
-| LEADER_ELECT | \-\-leader-elect | Start leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.|
+| DISABLE_LEADER_ELECTION | \-\-disable-leader-election | Start leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.|
```

Comment:
```suggestion
| DISABLE_LEADER_ELECTION | \-\-disable-leader-election | Disable the leader election client. Disable this when running a single replica components. (default = false)|
```

Reply:
Did we run `make docgen` on this? It should automatically remove these and update their descriptions IIRC